apps/src/lib/cli.rs

```diff
@@ -495,6 +521,9 @@ pub mod args {
         }));
     const LEDGER_ADDRESS_OPT: ArgOpt<tendermint::net::Address> =
         LEDGER_ADDRESS.opt();
+    const LEDGER_ADDRESS_ABOUT: &str =
```

Comment:
I don't see any definition of a `scheme` here. Although maybe it is hidden in this wall of consts. Maybe alphebetize for searching ease?

Reply:
good point, I've sorted them in 146ad7c